### PR TITLE
fix(events): fix calling of onload and onerror event handlers on script tags

### LINKED ISSUE #262: onerror and onload on script tags not firing
The issue described in https://github.com/krux/postscribe/issues/132 does not appear to be fixed in version 2.0.6. onload and onerrorevents do not appear to be firing on scripts written via Postscribe.

The simple test case found below fails to execute the documentDone function:
https://github.com/krux/postscribe/pull/133/commits/df34363fabfdc3fa3124dc78fdc373a82dea6148

### PR DESCRIPTION
**Pull request by sethyates**
previously we were stepping on any event handlers registered on the tags themselves

Fixes #262

### CODE REVIEW

File: src/write-stream.js [hunk 1]
```diff
@@ -454,7 +454,7 @@ export default class WriteStream {
   _onScriptDone(tok) {
     // Pop script and check nesting.
     if (tok !== this.scriptStack[0]) {
-      this.options.error({message: 'Bad script nesting or script finished twice'});
```
Comment (jnewman) [inline src/write-stream.js]: I'm skeptical about renaming this

Comment (jnewman) [inline src/write-stream.js]: Mainly because it's a major version change if we do

**Comment by jnewman** [comment]
👍 💯

File: src/write-stream.js (hunk 1 above)
Comment (sethyates) [inline src/write-stream.js]: It's actually wrong.  All other places are using `msg`.

**Pull request merged**